addon/components/frost-select.js

```diff
@@ -127,7 +126,7 @@ export default Component.extend({
   items (data, filter, onInput) {
     // If no data to filter we are done
     if (!data) {
-      return data
+      return []
```

Comment:
Downstream components require an array so this ensures we always provide an array.